homeassistant/components/knocki/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "knocki",
+  "name": "Knocki",
+  "codeowners": ["@joostlek", "@jgatto"],
```

Comment:
@jgatto Are you OK to be listed as code owner?

Reply:
@edenhaus the user should be @jgatto1! @joostlek could you confirm this? Thanks!

Reply:
Oh whoops, will update this